boards/build.rs

```diff
@@ -24,6 +24,9 @@ fn main() {
         panic!("Boards must provide a `layout.ld` link script file");
     }
 
+    println!("cargo:rustc-link-arg=-L{}", std::env!("CARGO_MANIFEST_DIR"));
```

Comment:
This is new, comment explaining its purpose? Something where many boards inside a workspace makes things weird?

Reply:
Done